vuepress/docs/next/docs/consume/entando-apis.md

```diff
@@ -63,20 +65,16 @@ http://[YOUR-HOST-NAME]/entando-de-app/api/swagger-ui.html
 
 2. Click on the Authorize button in the upper right corner.
 
-3. Enter the client id and secret and click Authorize.
+3. Enter the Client ID and Secret from above. Click Authorize.
```

Comment:
`Client Id` like in step 4??

Reply:
trying to minimize backticking... here backticked client id is the same as the literal words client ID the term. I don't think there would be any confusion.